README.md

```diff
@@ -9,19 +9,23 @@
 
 ### Install Prerequisites
 #### Install Git
- * If you want to use Git in the command line, run `sudo apt install git`
+ * If you want to use Git in the command line, run `sudo apt install git`, you can get [Git for Windows](https://git-scm.com/) or use `choco install git`
```

Comment:
Clarify platforms

Reply:
Something like "run `sudo apt install git` on Ubuntu or on Windows download git from XX or run XX in the command line if you have chocolatey"